GPT/gpt.py

```diff
@@ -163,42 +165,53 @@ def gpt_help():
         with open(file_path, 'r') as f:
             lines = f.readlines()[2:]
 
-        # Create a temporary HTML file and write the content to it
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".html") as f:
-            # Write the HTML header with CSS for dark mode, larger font size, text wrapping, and margins
-            f.write(b""" 
-            <html>
-            <head>
-                <style>
-                    body { 
-                        background-color: #282a36; 
-                        color: #f8f8f2; 
-                        font-family: Arial, sans-serif; 
-                        font-size: 18px; 
-                        margin: 100px; 
-                    }
-                    pre { 
-                        white-space: pre-wrap; 
-                        word-wrap: break-word; 
-                    }
-                </style>
-            </head>
-            <body>
-            <pre>
-            """)
-
-            # Write each line of the file, replacing newlines with HTML line breaks
-            for line in lines:
-                f.write((line.replace('\n', '<br>\n')).encode())
-
-            # Write the HTML footer
-            f.write(b"""
-            </pre>
-            </body>
-            </html>
-            """)
-
-            temp_filename = f.name
-
-        # Open the temporary HTML file in the web browser
-        webbrowser.open('file://' + os.path.abspath(temp_filename))
\ No newline at end of file
+        builder = HTMLbuilder.Builder()
+        builder.h1("Talon GPT Prompt List")
+        for line in lines:
+            if "##" in line:
+                builder.h2(line)
+            else:
+                builder.p(line)
+
+        builder.render() 
+    
+
+    def gpt_find_talon_commands(command_description:str):
+        """Search for relevant talon commands"""
+        command_list = ""
+        for ctx in registry.active_contexts():
+            items = ctx.commands.items()
+            for _, command in items:
+                raw_command = remove_wrapper(str(command))
```

Comment:
That should be removing the string wrapper

Reply:
It should be, but it did not.  I printed the request being sent and it didn't.  I was able to strip it with a regex:

```
def remove_wrapper(text: str):
    regex = r'[^"]+"([^"]+)"'
    match = re.search(regex, text)
    return match.group(1)
```